source/renderer/app/config/timingConfig.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 // All configuration values for timeouts / intervals should go here
-export const REPORT_ISSUE_TIME_TRIGGER = 5 * 60; // 5 minutes | unit: seconds
+export const REPORT_ISSUE_TIME_TRIGGER = 30; // 5 minutes | unit: seconds
```

Comment:
the comment claims 5 minutes still, but the code says 30 seconds

Reply:
@cleverca22 this was edited for debugging purposes. 5 min is the real value which we will revert to at the end...